webapp/public/locale/en/strings.json

```diff
@@ -240,15 +238,19 @@
     "unknown": "Uncategorized Suma charge"
   },
   "mobility": {
+    "activate_location_warning": "Activate Location",
+    "add_funds_warning": "Add money to your account before using suma's Mobility Services.",
     "do_not_park_intro": "You cannot park in this area. We may take action for repeated violations.",
     "do_not_park_title": "Please do not park here",
     "do_not_ride_intro": "You cannot ride in this area. We may take action for repeated violations.",
     "do_not_ride_title": "Please do not ride here",
     "end_trip": "End Trip",
     "intro": "Ride with enhanced privacy and better prices.",
     "locate_me": "Locate me",
+    "location_instructions_missing": "We could not find instructions to activate location services on your browser. Contact your administrator for more help.",
```

Comment:
Render in the help email or number here. If this is a thing we ever expect someone to see (and because we don't control their phones, someone will see it), it needs to be useful. There are some places, like private accounts, we can use language like this because no one will ever see it based on what they do/their phone (they'd only see it if we mess up).

Reply:
I was thinking that phone number would be better but I don't know that number yet. I added the email.